### DISCUSSION TITLE
It always complains to add authorization token

### DISCUSSION THREAD

**Discussion by sankaSanjeeva** [ardatan/feTS#399] [Q&A] [answered]
I have added an Authorization token using a plugin but it gives me type error requesting headers again. Is there any way to configure it?

![Screenshot 2023-07-27 at 19 20 58](https://github.com/ardatan/feTS/assets/44424604/f391a071-21e2-4c19-af5c-a59e75d3e81e)

**Comment by nbbaier** [comment]
I'm also wondering about this, ran into the same problem!

**Comment by thelinuxlich** [comment]
I took a stab at it https://github.com/ardatan/feTS/pull/852

**Comment by ardatan** [comment, marked as answer]
Fixed by;
https://github.com/ardatan/feTS/pull/852
